modules/onnx/1.17.0/presubmit.yml

```
@@ -0,0 +1,15 @@
+matrix:
+  platform:
+    - debian10
+    - ubuntu2004
+    - macos
+    - macos_arm64
+    - windows
```

```suggestion
  platform:
    - debian10
    - debian11
    - ubuntu2004
    - ubuntu2204
    - ubuntu2404
    - macos
    - macos_arm64
    - windows
```
Please add more platforms.